### ISSUE TITLE
Upgrade to libffi 3.3

### ISSUE THREAD

**Issue by atgreen** [node-ffi-napi/node-ffi-napi#47]
Libffi 3.3 was released a few weeks ago.  It would be nice to see you use the 3.3 release so we can get a better handle on node-ffi-napi local changes, and get better test coverage.

**Comment by addaleax** [comment]
I fully agree, but tbh, other than accepting PRs I can’t really put in much work at this point.
